packages/dataviews/src/types.ts

```diff
@@ -179,11 +186,15 @@ export type Fields< Item > = Field< Item >[];
 
 export type Data< Item > = Item[];
 
-export type DataFormControlProps< Item > = {
-	data: Item;
+export type DataFormControlProps<
+	Item,
+	SupportsBulkEditing extends boolean = true,
```

Comment:
I added this additional type to make it easier to handle within fields that don't support bulk editing, but maybe there is a better way?